opencti-platform/opencti-graphql/src/modules/attributes/internalObject-registrationAttributes.ts

```diff
@@ -240,6 +240,7 @@ const internalObjectsAttributes: { [k: string]: Array<AttributeDefinition> } = {
       multiple: false,
       upsert: false,
       isFilterable: false,
+      sortBy: { key: 'max_confidence', path: 'group_confidence_level.max_confidence' },
```

Comment:
I do not understand why using an object instead of just a string `group_confidence_level.max_confidence`. I think you already explain to me something similar for filters but I still found the thing confusing.

Reply:
I agree with you, right now it does not seem useful.
The idea was to be a bit future proof, allowing to define multiple sorting paths, each one with a unique key that the API/client could provide. But we're far from here : the client uses fixed enum, enum values that must be attribute keys. 

I realize I need to either put this as an array, to embrace the future approach, or just get rid of the object and just give a path like you suggest.
Julien's approach was more toward the first solution so I'll go for that.